client/src/utils/updateMyEvents.ts

```diff
@@ -0,0 +1,24 @@
+import type { Rsvp } from "../types/Rsvp";
+import type { MyEvents } from "../types/MyEvents";
+
+export const updateMyEvents = (rsvp: Rsvp) => {
```

Comment:
nit: Name seems more like a `addNewRSVPToMyEvents` to me (but I guess that's probably too wordy lol):


```suggestion
export const addNewRSVPToMyEvents = (rsvp: Rsvp) => {
```

Reply:
too wordy, its not like you struggle with that generally do you? 😆